src/cp.ts

```diff
@@ -1,7 +1,5 @@
-import fs from 'node:fs';
 import { WritableStreamBuffer } from 'stream-buffers';
-import * as tar from 'tar';
-import tmp from 'tmp-promise';
```

Comment:
Sorry for the back and forth - I think this is the only use of this dependency in the project. If it's removed here, it can be removed from package.json and package-lock.json too.

Reply:
Done (and removed `ws` also since it is not a direct dependency)